AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -40,6 +43,44 @@ module HierarchicalVersionUtils {
     return Success(Crypto);
   }
 
+  method createMdDigest (
```

Comment:
I know I am the fool who first called it an MD digest, 
but could I ask that we call it BKC;
while working on docs, 
it became apparent that we needed a different term.
```suggestion
  method createBKCDigest (
```

Reply:
Done